go.mod

```diff
@@ -65,18 +57,15 @@ require (
 	github.com/talos-systems/grpc-proxy v0.2.0
 	github.com/talos-systems/net v0.2.0
 	github.com/talos-systems/talos/pkg/machinery v0.0.0-20200818212414-6a7cc0264819
-	github.com/u-root/u-root v6.0.0+incompatible // indirect
-	github.com/vishvananda/netns v0.0.0-20200520041808-52d707b772fe // indirect
 	github.com/vmware-tanzu/sonobuoy v0.19.0
 	github.com/vmware/vmw-guestinfo v0.0.0-20200218095840-687661b8bd8e
-	go.etcd.io/etcd v0.5.0-alpha.5.0.20200819165624-17cef6e3e9d5 // v3.4.10
+	go.etcd.io/etcd v3.3.13+incompatible // v3.4.10
```

Comment:
this is a bad change, please don't move the tag there. it's a long story with Go modules, etcd and tags, but the tag should be kept as it is...

Reply:
Yeah, that's disturbing.  I didn't change that, at least not explicitly.  I _did_ run a `go mod tidy` because the build complained about `go.sum` being modified during the build.  My understanding of go modules is that it should never cross major change boundaries like that from a simple `tidy`.  Perhaps it is because of that version comment?  I have no idea what that is or if it is significant.

Reply:
Probably better to revert `go.mod`, and `go mod tidy` shouldn't touch it much